src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureKeyVaultReference/AzureKeyVaultSecretProvider.cs

```diff
@@ -61,6 +62,8 @@ public async Task<string> GetSecretValue(Uri secretUri, string key, Cancellation
                 if (client != null)
                 {
                     KeyVaultSecret secret = await client.GetSecretAsync(secretName, secretVersion, cancellationToken).ConfigureAwait(false);
+                    logger?.LogDebug(String.Format("{0}\"{1}\"", LoggingConstants.RefreshKeyVaultSecretChanged, secret.Name));
```

Comment:
Nit: does this need string.format? Can we log using concatenation just like the info log below?